bin/update_sanctions_csv

```diff
@@ -10,12 +10,19 @@ my $help;
 my $eu_token;
 my $eu_url;
 my $sanction_file;
+my $hmt_url;
+my $ofac_sdn_url;
+my $ofac_consolidated_url;
```

Comment:
I thought this card was just adding more filtering criteria, why are we adding new sources?

Reply:
These sources were there from the beginning. I'm just adding new  optional args in order to accelerate `xt/20_update.t` test script (the slow HM T source specifically).

Reply:
But they can be used for other aims.